app/content/blog/wiki/item-categories-ids-and-list.md

```
@@ -0,0 +1,221 @@
+89 Sage's Arms
+90 Registrable Miscellany
+105 Blue Mage's Arms
+91 Viper's Arms
+92 Pictomancer's Arms
+```
+
+# sub categories ids to main categories
+
+1. Arms
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)